### ISSUE TITLE
[RichText] Can't create a sub-element in the list

### ISSUE THREAD

**Issue by JeanBeaurepaire** [LuccaSA/lucca-front#4189]
### Bug description

When you press the tab key, which normally creates a sub-element in the list, it focus on the first element in the toolbar. 
However, you should be able to create a sub-list.
In Lexical (see second video), you use the Esc key to exit rich text mode and then the tab key to focus on the toolbar. Should we fix it this way?

https://github.com/user-attachments/assets/32d727b9-dd2b-4a93-8607-ed1942b434c5
https://github.com/user-attachments/assets/3f4fadd3-3d1e-41b5-8924-b62b2e1d6967

**Comment by vvalentin-lucca** [comment]
Perhaps consider detecting whether we are in a list and adopting specific behavior for the tab key in this case?
